CalorieTracker/CalorieTracker/Extensions/Notifications.swift

```diff
@@ -0,0 +1,14 @@
+//
+//  Notifications.swift
+//  CalorieTracker
+//
+//  Created by Craig Swanson on 2/23/20.
+//  Copyright © 2020 Craig Swanson. All rights reserved.
+//
+
+import Foundation
+
+extension Notification.Name {
```

Comment:
Yes! No stringly-typed code here. Good job making this extension to prevent simple errors down the road.